h/search/query.py

```diff
@@ -347,8 +347,12 @@ def __call__(self, search, _):
         return search.exclude("exists", field="deleted")
 
 
-class HiddenFilter:
-    """Return an Elasticsearch filter for filtering out moderated or NIPSA'd annotations."""
+class NipsaFilter:
```

Comment:
These now look very similar (Nipsa and Hidden) but the plan is to refactor Hidden to account for moderators next.